src/remoteRequest.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Remote Request question type plugin
+ */
+define([
+    'jquery',
+    'underscore',
+    'vellum/mugs',
+    'vellum/tree',
+    'vellum/util',
+    'vellum/widgets',
+    'vellum/core'
+], function (
+    $,
+    _,
+    mugs,
+    Tree,
+    util,
+    widgets
+) {
+    var REMOTE_REQUEST_TYPE = "RemoteRequest",
+        REMOTE_REQUEST_SECTIONS = [
+            {
+                slug: "main",
+                displayName: gettext("Basic"),
+                properties: [
+                    "nodeID",
+                    "url",
+                    "parameters",
+                    "comment",
+                ],
+            },
+        ],
+        RemoteRequest = util.extend(mugs.defaultOptions, {
+            typeName: gettext('Remote Request'),
+            dataType: REMOTE_REQUEST_TYPE,
+            icon: 'fa fa-globe',
+            isDataOnly: true,
+            isTypeChangeable: false,
+            supportsDataNodeRole: true,
+            writeDataNodeXML: function (xmlWriter, mug) {
+                mug.form.vellum.data.remoteRequest.requestMugs.push(mug);
+            },
+            spec: {
+                url: {
+                    lstring: gettext('URL'),
+                    visibility: 'visible',
+                    presence: 'optional',
+                    widget: widgets.text,
+                    help: gettext('The URL to be requested on form load.'),
+                },
+                parameters: {
+                    lstring: gettext('Parameters'),
+                    visibility: 'visible',
+                    presence: 'optional',
+                    widget: widgets.xPath,
+                    xpathType: "generic",
+                    serialize: mugs.serializeXPath,
+                    deserialize: mugs.deserializeXPath,
+                    help: gettext('A reference to a parameters node, "/data/request_params"'),
```

Comment:
This says it's a reference to a single node, but the input accepts any xpath expression. Can/should we do anything to limit what people put in this box?

Reply:
Also, should this say something like "Drag a question here" rather than talking about a node reference? I realize this is advanced functionality, but still.

Reply:
Unfortunately we don't have a widget type that only accepts a single ref, so I opted to use `widgets.xPath` since it is a nicer experience than `widgets.text` for drag/drop, etc. I don't think this use case is important enough to add a new "ref" widget type.

Good suggestion on the wording change.